slcan_attach.c

```diff
@@ -168,7 +170,15 @@ int main(int argc, char **argv)
 			write(fd, buf, strlen(buf));
 		}
 
-		if (send_open) {
+		/* prefer listen-only over normal open */
+		if (send_listen) {
+			sprintf(buf, "L\r");
+			write(fd, buf, strlen(buf));
+
+			if (send_open)
+				printf("ignoring -o, due to listen-only option -l\n");
+
```

Comment:
Nitpick please remove this extra line. If there are no further objections, I'll do this while applying the patch.